sdk/python/lib/test/langhost/output_property_dependencies/__main__.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2016-2018, Pulumi Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import functools
+import pulumi
+from typing import Optional
+
+@pulumi.input_type
+class MyResourceArgs:
+    def __init__(__self__, *, in_prop: Optional[pulumi.Input[str]] = None):
+        if in_prop is not None:
+            pulumi.set(__self__, "in_prop", in_prop)
+
+    @property
+    @pulumi.getter(name="inProp")
+    def in_prop(self) -> Optional[pulumi.Input[str]]:
+        return pulumi.get(self, "inProp")
```

Comment:
This suggestion doesn't strictly matter since I don't believe the getter is actually called by this test, and only the decorator is necessary for the translation, but the implementation should be:

```suggestion
        return pulumi.get(self, "in_prop")
```